### ISSUE TITLE
[Bug] First-step Loss NaN when enabling deterministic mode via msprobe.seed_all in Wan2.1 training

### ISSUE THREAD

**Issue by zbian99** [ByteDance-Seed/VeOmni#306]
### Description

When training the **Wan2.1** model, enabling deterministic mode using `msprobe.pytorch.seed_all(mode=True)` causes the training loss to become `NaN` at the very first step.

The training proceeds normally if `seed_all` is not used or if deterministic mode is disabled.

### Reproduction Steps

Add the following code snippet to the training initialization script:

```python
from msprobe.pytorch import seed_all

# Enabling deterministic mode triggers the issue
seed_all(mode=True)

# ... Proceed to Wan2.1 model initialization and training loop ...
```

**Observed Behavior:**

  * With `seed_all(mode=True)`: Loss is `NaN` at Step 1.
  * Without `seed_all`: Loss is normal.

### Environment Information

  * **Model:** Wan2.1 i2v 14B
  * **Hardware:** A100
  * **PyTorch:** 2.8.0+cu128

**Comment by yuyu5333** [comment]
@zbian99  Thank you for your discovery. I will try to reproduce it recently.
